### PR TITLE
Use the ubiquitous language for daily debit limit

### LINKED ISSUE #81: Check for use of ubiquitous language in daily debit limit events.
Looks like there's some different terminology "consumed" vs "used", etc.

### PR DESCRIPTION
**Pull request by koden-km**
#### What change does this introduce?

This PR fixes some of the terminology used in the daily debit limit implementation to match the ubiquitous language.

#### What issues does this relate to?

Fixes #81 

#### Why make this change?

The language used in the domain should match the ubiquitous language as much as possible so that the domain experts and programmers understand each other.

#### Is there anything you are unsure about?

No.

### CODE REVIEW

File: domain/dailydebitlimit.go [hunk 1]
```diff
@@ -14,8 +14,9 @@ const maximumDailyDebitLimit = 900000
 // dailyDebitLimit is the aggregate root for an account daily debit limit
 // policy.
 type dailyDebitLimit struct {
-	// UsedAmount is the sum of debit amounts used during the period, in cents.
-	UsedAmount int64
+	// ConsumedAmount is the sum of debit amounts consumed during the period, in
```
Comment (danilvpetrov) [inline domain/dailydebitlimit.go:17]: "during the period" sounds like a certain period is implied, perhaps the period should be mentioned? E.g. `ConsumedAmount is the sum of debit amounts consumed during the daily period...`.  If there is no need to emphasise any particular period, perhaps it should sound like `ConsumedAmount is the sum of debit amounts consumed during a certain period...`

Also, I think you meant to use the amount in singular: 

```suggestion
	// ConsumedAmount is the sum of debit amount consumed during the period, in
```

Comment (danilvpetrov) [reply domain/dailydebitlimit.go:17]: Sorry I did not consider the top comment for the `dailyDebitLimit ` struct which mentions the daily limit and hence the daily period, you can disregard my first paragraph in the comment.

File: domain/dailydebitlimit.go [hunk 2]
```diff
@@ -14,8 +14,9 @@ const maximumDailyDebitLimit = 900000
...
+	// cents.
+	ConsumedAmount int64
```
Comment (jmalloc) [inline domain/dailydebitlimit.go:19]:
```suggestion
	// TotalDebitsForDay is the total of all debits for the day, in cents.
	TotalDebitsForDay int64
```

File: domain/dailydebitlimit.go [hunk 3]
```diff
@@ -26,7 +27,7 @@ func (d *dailyDebitLimit) Consume(s dogma.AggregateCommandScope, m commands.Cons
 			DebitType:     m.DebitType,
 			Amount:        m.Amount,
 			Date:          m.ScheduledDate,
-			LimitUsed:     d.UsedAmount,
+			LimitConsumed: d.ConsumedAmount,
 			LimitMaximum:  maximumDailyDebitLimit,
```
Comment (jmalloc) [inline domain/dailydebitlimit.go:31]:
```suggestion
			TotalDebitsForDay: d.TotalDebitsForDay,
			DailyLimit:  maximumDailyDebitLimit,
```

**Review by jmalloc — CHANGES_REQUESTED** [review]
I discussed the naming with @koden-km in person, and we came up with the changes within my review.

**Review by jmalloc — APPROVED** [review]

**Review by danilvpetrov — COMMENTED** [review]
Looking good, sorry for a late review.

**Pull request merged**